types/domain_type.go

```diff
@@ -0,0 +1,106 @@
+package types
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/pkg/errors"
+)
+
+// NetworkID are intended to separate different SSV networks. A network can have many forks in it.
+type NetworkID [1]byte
+
+var (
+	MainnetNetworkID = NetworkID{0x0}
```

Comment:
When we do spec rethink we can move it all to config

Reply:
IMO this is something that shouldn't be changed often hence it can be part of the code, as a hard-coded config, this is how it's done at least in other clients. though it introduces an issue of being less dynamic like creating a specific network config for a on-demand spun-up test-network, but I assume we can solve it with adding a test network id.